### PR TITLE
Start current release file

### PR DESCRIPTION
**Pull request by stefpiatek**
Will ask for a review once we're getting close to the release

### CODE REVIEW

File: changelogs/2022-02-changelog.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+### Changes/fixes
+
+* Temporal columns are now all timezone aware, this has been enforced with automated testing of any timezone field
+* Hl7 feed only parses new patient infections
+* Patient infections that are associated with an encounter have a hospital_visit_id
+* Improved speed of load times by updating caching
+* Fixes Log4j exploits
+* Coded flowsheets are now parsed as the raw id values given in the Hl7 feed
+
```
Comment (stefpiatek) [inline changelogs/2022-02-changelog.md:16]: oh and we need changes to visit_obvervation now having a source_system column. Then have changes to visit_observation_type columns. Processing of flowsheets now only has one visit_observation_type for both the hl7 and caboodle types.

Comment (AnikaC-git) [reply changelogs/2022-02-changelog.md:16]: Should we say here that we therefore maintain two identifiers? it used to be just one ...

**Review by AnikaC-git — COMMENTED** [review]
Thanks a lot for adding this in. Just have a minor suggestion add to Stef's comment...

File: changelogs/2022-02-changelog.md (hunk 1 above)
Comment (stefpiatek) [reply changelogs/2022-02-changelog.md:16]: Oh yeah that wasn't a suggestion of wording to put in, just an overview of what was missing!

**Review by AnikaC-git — APPROVED** [review]
Looks great to me! 🥇

**Pull request merged**